pkg/distributor/distributor_test.go

```diff
@@ -369,7 +369,7 @@ func TestDistributor_PushHAInstances(t *testing.T) {
 						KVStore:         kv.Config{Mock: mock},
 						UpdateTimeout:   100 * time.Millisecond,
 						FailoverTimeout: time.Second,
-					})
+					}, prometheus.NewRegistry())
```

Comment:
```suggestion
					}, nil)
```

Since we don't verify metrics.